services/streammanager/googlepubsub/googlepubsubmanager.go

```diff
@@ -14,6 +15,7 @@ import (
 	"google.golang.org/grpc/credentials/insecure"
 	"google.golang.org/grpc/status"
 
+	conf "github.com/rudderlabs/rudder-go-kit/config"
```

Comment:
We mey rename the current config variable to conf and leave this import without an alias